src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs

```diff
@@ -67,6 +67,31 @@ private protected virtual SimpleResolver CreateResolver(PowerFxConfig alternateC
         public IReadOnlyList<Token> Tokenize(string expressionText)
             => TexlLexer.LocalizedInstance.GetTokens(expressionText);
 
+        /// <summary>
+        /// Check only syntax of the expression.
+        /// </summary>
+        /// <param name="expressionText"></param>
+        /// <returns></returns>
+        // TODO: This should be done differently, temporary hack!
```

Comment:
As discussed earlier we should have a separate parse-only result. This is a temporary hack that should be done properly.

Reply:
See https://github.com/microsoft/Power-Fx/pull/319 for making ParseResult public.

Reply:
Awesome, this works perfectly.